src/qdrant_client/conversions/vectors_config.rs

```diff
@@ -0,0 +1,16 @@
+use crate::qdrant::vectors_config::Config;
```

Comment:
Shouldn't this file rather be placed in `grpc_conversions` or is there a reason you've put it here?

Reply:
Actually, I would prefer to move conversion out of those `primitives.rs` instead